pkgs/tools/misc/tewisay/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildGoPackage, fetchgit, asciidoc, go-runewidth, pflag }:
```

Comment:
Your package fails to build.
You have to specify Go dependencies by using the `goDeps` attributes. See   https://nixos.org/nixpkgs/manual/#ex-buildGoPackage-3
These dependencies can be generated by some tools such as go2nix.